playbooks/deploy_cluster_40.yml

```diff
@@ -56,6 +56,37 @@
     vars:
       excluded_services:
       - progress.service
+  - name: Wait for MCS endpoint to show up
+    uri:
+      url: "{{ mcd_endpoint }}/config/master"
+      validate_certs: false
+    delay: 10
+    retries: 60
+    register: mcs
+    until:
+    - "'status' in mcs"
+    - mcs.status == 200
+    ignore_errors: true
+  - when: mcs is failed
+    block:
+    - name: Get node logs
+      command: journalctl --no-pager -u bootkube
+      register: bootkube_logs
+      ignore_errors: true
+    - name: Collect a list of containers
+      command: crictl ps -a -q
```

Comment:
silly question - if you running using RHCOS image, then i suspect you need sudo and if so do you need it here ? or you have  that covered in ansible.cfg maybe ?

Reply:
This playbook won't work on RHCOS - a usual installer should be used instead.

The playbook assumes root account or sudo with `become` on target hosts